runtime/oti/j9nonbuilder.h

```diff
@@ -44,6 +44,7 @@
 #define CLASSNAME_INVALID			0
 #define CLASSNAME_VALID_NON_ARRARY	1
 #define CLASSNAME_VALID_ARRARY		2
+#define CLASSNAME_VALID				3
```

Comment:
This needs documenting to show that it is a bitfield. To reinforce this, CLASSNAME_VALID should be the bit OR of the other two values, rather than being defined directly to 3.

Reply:
Updated against the suggested.